org.eclipse.scanning.event/src/org/eclipse/scanning/event/ConsumerImpl.java

```diff
@@ -297,7 +300,8 @@ public void setRunner(IProcessCreator<U> runner) throws EventException {
 	
 	@Override
 	public void start() throws EventException {
-				
+		
+		latchStart = new CountDownLatch(1);
 		final Thread consumerThread = new Thread("Consumer Thread "+getName()) {
 			public void run() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)